solidity/contracts/converter/types/liquidity-pool-v1/LiquidityPoolV1Converter.sol

```diff
@@ -297,6 +297,20 @@ contract LiquidityPoolV1Converter is LiquidityPoolConverter {
         removeLiquidityFromPool(reserveTokens, reserveMinReturnAmounts, totalSupply, _amount);
     }
 
+    /**
+      * @dev calculates the amount of each reserve token entitled for a given amount of pool tokens
+      *
+      * @param _amount          amount of pool tokens
+      * @param _reserveTokens   address of each reserve token
+      *
+      * @return the amount of each reserve token entitled for the given amount of pool tokens
+    */
+    function removeLiquidityReturn(uint256 _amount, IERC20Token[] memory _reserveTokens) public view returns (uint256[] memory) {

```

Comment:
you sure the gas savings here are worth the extra function?

Reply:
Two external calls, so yes.

Reply:
ok